source/features/remove-label-faster.css

```
@@ -19,6 +19,10 @@
 	fill: rgb(var(--label-r) var(--label-g) var(--label-b));
 }
 
+[data-color-mode='dark'][data-dark-theme*='dark'] .rgh-remove-label-faster:is(:focus, :hover) svg {
```

I think this part isn't needed. The variable probably already matches the theme
```suggestion
.rgh-remove-label-faster:is(:focus, :hover) svg {
```